src/app/components/TutorialDialog/component/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import background from 'assets/images/tutorial/test.svg';
+import close from 'assets/images/tutorial/close.svg';
+import { Screen1 } from '../screen1';
+import { Screen2 } from '../screen2';
+import { Screen3 } from '../screen3';
+
+import { translations } from 'locales/i18n';
+import { useTranslation } from 'react-i18next';
+
+export function TutorialDialogComponent(props) {
+  const { t } = useTranslation();
+  const [mouseLeave, setMouseLeave] = useState(false);
+  const activeTutorial =
+    window.localStorage.getItem('tutorial_active') === 'true' &&
+    props.onNetwork === true
+      ? true
+      : false;
+  const [screen, setScreen] = useState(activeTutorial ? 2 : 1);
+
+  function changeScreen(num) {
+    setScreen(num);
+  }
+
+  function back() {
+    setScreen(1);
+  }
+
+  return (
+    <>
+      <div
+        className="wallet-tutorial_container position-absolute mx-auto"
+        onMouseLeave={() => {
+          console.log('Mouse out');
+          if (screen === 2) {
+            setMouseLeave(true);
+          } else {
+            setMouseLeave(false);
+          }
+        }}
+      >
+        <div className={`${screen === 2 && 'left'} position-absolute`}>
+          <div className="background position-relative w-100">
+            <img src={background} alt="" className="w-100 h-100" />
+          </div>
+          <div className="close" onClick={props.handleClose}>
+            <img src={close} alt="close" />
+          </div>
+          {screen !== 1 && (
+            <div className="back position-absolute" onClick={() => back()}>
+              <button>Back</button>
```

Comment:
Move word "Back" to translations file, maybe use common namespace for key to eg. `translations.common.back`